web_ui/eslint.config.mjs

```diff
@@ -175,7 +175,7 @@ export default [
                 'error',
                 'ignorePackages',
                 {
-                    js: 'never',
+                    js: 'always',
```

Comment:
`day-js` is not exported as module so it's treated as CommonJS, therefore, we need to add ".js" to the imports